src/current/_data/releases.yml

```
@@ -6124,3 +6124,37 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v23.1.21
+
+- release_name: v24.1.1
+  major_version: v24.1
+  release_date: '2024-06-13'
+  release_type: Production
+  go_version: go1.22.0
+  sha: 25e4a522b5a77449e43404987337d75f53e66304
+  has_sql_only: true
+  has_sha256sum: true
+  mac:
+    mac_arm: true
+    mac_arm_experimental: true
+    mac_arm_limited_access: true
+  windows: true
+  linux:
+    linux_arm: true
+    linux_arm_experimental: false
+    linux_arm_limited_access: false
+    linux_intel_fips: true
+    linux_arm_fips: false
+  docker:
+    docker_image: cockroachdb/cockroach
+    docker_arm: true
+    docker_arm_experimental: false
+    docker_arm_limited_access: false
+  source: true
+  previous_release: v24.1.0
+  cloud_only: true
+  cloud_only_message_short: 'Available only to select CockroachDB Cloud clusters'
+  cloud_only_message: >
+      This version is currently available only to select
+      CockroachDB Cloud clusters. To request to upgrade
+      a CockroachDB Self-Hosted cluster to this version,
+      [contact Support](https://support.cockroachlabs.com/hc/en-us/requests/new).
```

```suggestion
      [contact Support](https://support.cockroachlabs.com/hc/requests/new).
```
If the wiki instructions have this bit, can you please update them? This will force folks to the English Microsoft site, regardless of their locale. If you access the link without it, you'll see that it gets added automatically because of your locale.

---

I copy-pasted this from a previous Cloud-first entry. Will amend the rest on this page.